metadata.json

```diff
@@ -69,7 +77,7 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": ">= 3.8.1 < 2015.3.0"
+      "version_requirement": ">= 2015.3.0 < 2016.1.0"
```

Comment:
This module requirement applies to where it's being run, which could be as part of `puppet apply`. That could still include `3.8.1`, so I'm not sure why the minimum is changing.